public/app/features/alerting/unified/components/rule-viewer/tabs/Details.tsx

```diff
@@ -79,12 +79,12 @@ const Details = ({ rule }: DetailsProps) => {
           {hasEvaluationDuration && (
             <>
               Last evaluation
-              {evaluationTimestamp && evaluationDuration && (
+              {evaluationTimestamp && evaluationDuration ? (
```

Comment:
This was an incidental bug I found, where it would render the last evaluation as `0`, due to the rendering for falsy values

Reply:
Good catch!